packages/pulpcore-obsolete-packages/pulpcore-obsolete-packages.spec

```diff
@@ -1,13 +1,16 @@
 Name: pulpcore-obsolete-packages
-Version: 1.1.0
+Version: 1.2.0
 Release: 1%{?dist}
 License: MIT
 Summary: A package to obsolete retired packages
 URL: https://github.com/theforeman/pulpcore-packaging
 BuildArch: noarch
 
 Obsoletes:      python3-django-currentuser < 0.5.3-6
+Obsoletes:      python3.11-pymongo
+Obsoletes:      python3.11-mongoengine
 %if 0%{?rhel} == 8
+Obsoletes:      python3-pulp-2to3-migration < 0.12.0-3
```

Comment:
I was confused at first, but there were really no other names of this pkg:
```
[evgeni@web01 pulpcore]$ find . -name \*2to3\*noarch.rpm | xargs -n1 basename | sed 's/-0\..*//' | sort -u
python3-pulp-2to3-migration
```

Reply:
Technically, there is one el9 build:
```
[evgeni@web01 pulpcore]$ find . -name \*2to3\*el9\*noarch.rpm
./3.18/el9/x86_64/python3-pulp-2to3-migration-0.12.0-1.el9.noarch.rpm
```
but we never supported that release on el9 for Katello, so we don't need to care, IMHO